parkbenchcommon/tmpfs.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2017-2018 Joel Allen Luellwitz and Andrew Klapp
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+""" Provides helper functions for tmpfs usage."""
+
+__all__ = ['mount_tmpfs', 'path_is_tmpfs_mountpoint', 'TmpfsMountError']
+
+import os
+import subprocess
+
+class TmpfsMountError(Exception):
+    """ Raised when a tmpfs mount operation appears to fail."""
+
+def path_is_tmpfs_mountpoint(path):
+    """ Checks that a path is mounted as tmpfs.
+
+    Returns True if it is mounted, False if it is not.
+
+    path: The path to check
+    """
+    path = os.path.abspath(path)
+
+    return 'none on %s type tmpfs' % path in str(subprocess.check_output('mount'))
+
+def mount_tmpfs(path, size):
```

Comment:
I'd probably call this mount_ramfs or mount_ramdisk because tmpfs is a specific kind of ramdisk and I don't think that detail is important to our consumers.